### ISSUE TITLE
Load translations

### ISSUE THREAD

**Issue by bagjunggyu** [linuxmint/indicator-sound#1]
We don't have translations now for indicator-sound.
I think we need them.

**Comment by monsta** [comment]
Check if you have `language-pack-gnome-ko-base` installed - Ubuntu developers put indicator translations in such packages.
For example, I have Russian translation for indicator-sound from `language-pack-gnome-ru-base`.

**Comment by bagjunggyu** [comment]
Thanks, I can see the same "indicator-sound.mo" in Sylvia Cinnamon now,
but, It was reported since Linux Mint Xfce has no translated sound applet? from panel.
Thus, Clem suggested me to report this issue.
I wish I can see it as translated in the Sylvia Xfce sound applet.
As I am seeing this translated indicator-sound.mo file here.

**Comment by monsta** [comment]
It should be translated in previous Mint Xfce versions too, as the Ubuntu base is the same. You can check it in 18.2, for example.

**Comment by bagjunggyu** [comment]
But, It doesn't show me in the panel , it shows English

**Comment by bagjunggyu** [comment]
I've checked that , I think that applet using in Mint Xfce seems not indicator-sound in terms of view of translation file indicator-sound.mo, please see this screenshot.
[](https://drive.google.com/file/d/147QIu9gkaM73hr4tgYrdH7BWWdjl2nvR/view?usp=sharing)

Contents of sound indicator in the panel are not translated and also its contents(translated contents) are different from indicator-sound.mo , it is some other thing.
I checked its contents with converting _indicator-sound.mo_ file to _indicator-sound.po_ file

**Comment by monsta** [comment]
But that's a screenshot of [pavucontrol](https://screenshots.debian.net/package/pavucontrol) app...

**Comment by bagjunggyu** [comment]
Good to know its name :)
But, when clicking speaker like icon indicator in the panel (which I knew until now as sound indicator), 
it brings me to that applet **pavucontrol** , and it is like what you have seen above screenshot.
What should I do for fixing it?

**Comment by monsta** [comment]
The panel icon is from indicator-sound, it just launches pavucontrol to configure sound stuff.

I found out that pavucontrol translations are done on [Transifex](https://www.transifex.com/lennart/pavucontrol/).

**Comment by bagjunggyu** [comment]
Thank you :)
I found out that Korean is missing from the language list. 
I would request Korean translation to project manager.
I wish I could translate it as fast as I can, before release of Sylvia Xfce.
I am rushing from now on :)

**Comment by monsta** [comment]
No, don't rush... the problem is that you're translating the latest upstream version, which might go into the current development release of Ubuntu (18.04). Latest translations won't go into Ubuntu 16.04 on which Mint 18.x releases are based. But Mint 19.x releases will be based on Ubuntu 18.04, so they'll have your translations.

Anyway, you can close this report as it's not a Mint issue.

**Comment by bagjunggyu** [comment]
Unfortunately, I can not help it.
Sure, I Would close this issue.
Thank you for all of it.

**Comment by bagjunggyu** [comment]
They don't seem to inactivate.
Can you handle this?
https://github.com/pulseaudio/pavucontrol/pull/2/files

**Comment by bagjunggyu** [comment]
I waited for their answer for the request of Korean team https://www.transifex.com/lennart/pavucontrol/
,but there is no answer until now.
So that, I PR to their github - https://github.com/pulseaudio/pavucontrol/pull/2/files,
But, it has turned out that it is just mirror.
And by help of some people, I made to report to bugzilla
https://bugs.freedesktop.org/show_bug.cgi?id=104476
and have deleted PR.
Well, it is the full story to enable to see translated to Korean pavucontrol on Tara :)
